site/layouts/_default/baseof.html

```diff
@@ -5,6 +5,7 @@
   </head>
   {{ block "body_override" . }}<body>{{ end }}
     {{ partial "skippy" . }}
+    {{ partial "icons" . }}
```

Comment:
Does the sprite include only the needed icons? If so, would be nice if we had a quick command documented to generate them in the future.

Reply:
I manually created this, copying and pasting and editing from the Bootstrap Icons. It does only include icons that are needed across the homepage and docs layouts. Not much is wasted between the two right now, and I'll likely add even more icons to the homepage in another commit.